datafusion-examples/examples/proto/composed_extension_codec.rs

```diff
@@ -62,17 +63,19 @@ pub async fn composed_extension_codec() -> Result<()> {
         Arc::new(ParentPhysicalExtensionCodec {}),
         Arc::new(ChildPhysicalExtensionCodec {}),
     ]);
+    let proto_codec = DefaultPhysicalExtensionProtoCodec {};
```

Comment:
could we keep original methods which would create DPEPC{} internally to keep backward compatibility and add new methods which would accept DPEPC as a parameter?